src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -110,14 +112,22 @@ int currentGroupLevel(const Opm::Schedule& sched, const Opm::Group& group, const
     return level;
 }
 
+
 void groupProductionControllable(const Opm::Schedule& sched, const Opm::SummaryState& sumState, const Opm::Group& group, const size_t simStep, bool& controllable)
 {
     using wellCtrlMode   = ::Opm::RestartIO::Helpers::VectorItems::IWell::Value::WellCtrlMode;
     if (controllable)
         return;
 
-    for (const auto& group_name : group.groups())
-        groupProductionControllable(sched, sumState, sched.getGroup(group_name, simStep), simStep, controllable);
+    for (const auto& group_name : group.groups()) {
```

Comment:
The semantics of this function is to go all the way *down* to the well level - and if at least one of the wells is *currently* under group control we return true. Is that correct?

Reply:
I agree that this relates to a group currently being controllable, I therefore change the name to groupCurrentlyProductionControllable and correspondlingly to groupCurrentlyInjectionControllable for group injection. The requirement is that there are at least one well that is currently controllable belonging to a group that is controllable from a higher level.